<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLoginWidget</class>
 <widget class="QWidget" name="CLoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="CStackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="CLoginInfoWidget">
    <widget class="QLabel" name="CLblBackgroundUp_Login">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>400</width>
       <height>150</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="CLblBackgroundDown_Login">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>400</width>
       <height>150</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="CBtnClose_Login">
     <property name="geometry">
      <rect>
       <x>376</x>
       <y>10</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="CBtnLogin_Login">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>250</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="CBtnMinimum_Login">
     <property name="geometry">
      <rect>
       <x>352</x>
       <y>10</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="CBtnSwitch_Login">
     <property name="geometry">
      <rect>
       <x>328</x>
       <y>10</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="CChkAutoLogin_Login">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>220</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="CChkRememberPasswd_Login">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>220</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="CCmbUser_Login">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="CLblPasswd_Login">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="CLblUser_Login">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="CLedPasswd_Login">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>190</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="CBtnRegistAccount_Login">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>160</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="CBtnModifyPasswd_Login">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>190</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="CServerInfoWidget">
    <widget class="QPushButton" name="CBtnCancel_Server">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>250</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="CBtnClose_Server">
     <property name="geometry">
      <rect>
       <x>376</x>
       <y>10</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="CBtnConfirm_Server">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>250</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="CBtnMinimum_Server">
     <property name="geometry">
      <rect>
       <x>352</x>
       <y>10</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="CBtnSwitch_Server">
     <property name="geometry">
      <rect>
       <x>328</x>
       <y>10</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="CCmbIp_Server">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="CLblBackgroundDown_Server">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>400</width>
       <height>150</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="CLblBackgroundUp_Server">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>400</width>
       <height>150</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="CLblIp_Server">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="CLblPort_Server">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>70</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="CLedPort_Server">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>190</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <zorder>CLblBackgroundUp_Server</zorder>
    <zorder>CLblBackgroundDown_Server</zorder>
    <zorder>CBtnCancel_Server</zorder>
    <zorder>CBtnClose_Server</zorder>
    <zorder>CBtnConfirm_Server</zorder>
    <zorder>CBtnMinimum_Server</zorder>
    <zorder>CBtnSwitch_Server</zorder>
    <zorder>CCmbIp_Server</zorder>
    <zorder>CLblIp_Server</zorder>
    <zorder>CLblPort_Server</zorder>
    <zorder>CLedPort_Server</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
